From: Michael Schutte <michi@debian.org>
Subject: [PATCH] fixes/manpages

Some manpage corrections:

 * Remove .IX lines.
 * Fix a groff syntax error in splitfont.1.
 * Correct a typo in mk_modmap.8, kindly pointed out by lintian.
 * Document that “openvt -e” has to be a session leader, closes:
   #610373.

Signed-off-by: Michael Schutte <michi@debian.org>

---
 man/man1/dumpkeys.1.in      |    1 -
 man/man1/kbd_mode.1         |    1 -
 man/man1/loadkeys.1.in      |    1 -
 man/man1/openvt.1           |    7 +++++++
 man/man1/psfaddtable.1      |    1 -
 man/man1/psfgettable.1      |    1 -
 man/man1/psfstriptable.1    |    1 -
 man/man1/psfxtable.1        |    1 -
 man/man1/setleds.1          |    1 -
 man/man1/setmetamode.1      |    1 -
 man/man1/showkey.1          |    1 -
 man/man1/unicode_start.1    |    1 -
 man/man1/unicode_stop.1     |    1 -
 man/man1misc/splitfont.1    |    2 +-
 man/man5/keymaps.5          |    5 -----
 man/man8/setfont.8.in       |    1 -
 man/man8misc/mk_modmap.8    |    4 ++--
 man/man8misc/setvesablank.8 |    1 -
 18 files changed, 10 insertions(+), 22 deletions(-)

diff --git a/man/man1/dumpkeys.1.in b/man/man1/dumpkeys.1.in
index 00f9e66..eef8edb 100644
--- a/man/man1/dumpkeys.1.in
+++ b/man/man1/dumpkeys.1.in
@@ -11,7 +11,6 @@ dumpkeys \- dump keyboard translation tables
 .BI \-\-charset= charset
 ]
 .SH DESCRIPTION
-.IX "dumpkeys command" "" "\fLdumpkeys\fR command"  
 .LP
 .B dumpkeys
 writes, to the standard output, the current contents of the keyboard
diff --git a/man/man1/kbd_mode.1 b/man/man1/kbd_mode.1
index 960b36c..a3f061f 100644
--- a/man/man1/kbd_mode.1
+++ b/man/man1/kbd_mode.1
@@ -10,7 +10,6 @@ kbd_mode \- report or set the keyboard mode
 .I -C CONSOLE
 ]
 .SH DESCRIPTION
-.IX "kbd_mode command" "" "\fLkbd_mode\fR command"  
 .LP
 Without argument,
 .B kbd_mode
diff --git a/man/man1/loadkeys.1.in b/man/man1/loadkeys.1.in
index e39eb76..20b01cb 100644
--- a/man/man1/loadkeys.1.in
+++ b/man/man1/loadkeys.1.in
@@ -31,7 +31,6 @@ loadkeys \- load keyboard translation tables
 ]
 .LP
 .SH DESCRIPTION
-.IX "loadkeys command" "" "\fLloadkeys\fR command"  
 .LP
 The program
 .B loadkeys
diff --git a/man/man1/openvt.1 b/man/man1/openvt.1
index 9a58a80..c58f1e5 100644
--- a/man/man1/openvt.1
+++ b/man/man1/openvt.1
@@ -28,6 +28,13 @@ Force opening a VT without checking whether it is already in use.
 Directly execute the given command, without forking.
 This option is meant for use in
 .IR /etc/inittab .
+If you want to use this feature in another context, be aware that
+.B openvt
+has to be a session leader in order for \-e to work.  See
+.BR setsid (2)
+or
+.BR setsid (1)
+on how to achieve this.
 .TP
 .I "\-s"
 Switch to the new VT when starting the command. The VT of the new command
diff --git a/man/man1/psfaddtable.1 b/man/man1/psfaddtable.1
index ad5c844..611d9bb 100644
--- a/man/man1/psfaddtable.1
+++ b/man/man1/psfaddtable.1
@@ -6,7 +6,6 @@ psfaddtable \- add a Unicode character table to a console font
 .B psfaddtable
 .I fontfile tablefile outfile
 .SH DESCRIPTION
-.IX "psfaddtable command" "" "\fLpsfaddtable\fR command"  
 .LP
 .B psfaddtable
 takes a console font in .psf format given by
diff --git a/man/man1/psfgettable.1 b/man/man1/psfgettable.1
index 228be18..5d8e572 100644
--- a/man/man1/psfgettable.1
+++ b/man/man1/psfgettable.1
@@ -7,7 +7,6 @@ psfgettable \- extract the embedded Unicode character table from a console font
 .I fontfile
 .RI [ outfile ]
 .SH DESCRIPTION
-.IX "psfgettable command" "" "\fLpsfgettable\fR command"  
 .LP
 .B psfgettable
 extracts the embedded Unicode character table from a .psf format
diff --git a/man/man1/psfstriptable.1 b/man/man1/psfstriptable.1
index f7e059d..b11ed4d 100644
--- a/man/man1/psfstriptable.1
+++ b/man/man1/psfstriptable.1
@@ -6,7 +6,6 @@ psfstriptable \- remove the embedded Unicode character table from a console font
 .B psfstriptable
 .I fontfile outfile
 .SH DESCRIPTION
-.IX "psfstriptable command" "" "\fLpsfstriptable\fR command"  
 .LP
 .B psfstriptable
 reads a .psf format console font from 
diff --git a/man/man1/psfxtable.1 b/man/man1/psfxtable.1
index adc3bc9..5cc89ad 100644
--- a/man/man1/psfxtable.1
+++ b/man/man1/psfxtable.1
@@ -14,7 +14,6 @@ psfxtable \- handle Unicode character tables for console fonts
 .IR outtable ]
 .RB [ \-nt ]
 .SH DESCRIPTION
-.IX "psfxtable command" "" "\fLpsfxtable\fR command"  
 .LP
 .B psfxtable
 handles the embedded Unicode character table for .psf format
diff --git a/man/man1/setleds.1 b/man/man1/setleds.1
index f484fa0..f9cd659 100644
--- a/man/man1/setleds.1
+++ b/man/man1/setleds.1
@@ -7,7 +7,6 @@ setleds \- set the keyboard leds
 .RI [ "-v" "] [" "-L" "] [" "-D" "] [" "-F" ]
 .RI [ {+|-}num "] [" {+|-}caps "] [" {+|-}scroll ]
 .SH DESCRIPTION
-.IX "setleds command" "" "\fLsetleds\fR command"  
 .LP
 .B Setleds
 reports and changes the led flag settings of a VT
diff --git a/man/man1/setmetamode.1 b/man/man1/setmetamode.1
index 2a0dd81..fdad981 100644
--- a/man/man1/setmetamode.1
+++ b/man/man1/setmetamode.1
@@ -8,7 +8,6 @@ setmetamode \- define the keyboard meta key handling
 .I {meta|bit|metabit | esc|prefix|escprefix}
 ]
 .SH DESCRIPTION
-.IX "setmetamode command" "" "\fLsetmetamode\fR command"  
 .LP
 Without argument,
 .B setmetamode
diff --git a/man/man1/showkey.1 b/man/man1/showkey.1
index 654e019..ad63434 100644
--- a/man/man1/showkey.1
+++ b/man/man1/showkey.1
@@ -5,7 +5,6 @@ showkey \- examine the codes sent by the keyboard
 .SH SYNOPSIS
 showkey [\-h|\-\-help] [\-a|\-\-ascii] [\-s|\-\-scancodes] [\-k|\-\-keycodes]
 .SH DESCRIPTION
-.IX "showkey command" "" "\fLshowkey\fR command"  
 .LP
 .B showkey
 prints to standard output either the scan codes or the keycode
diff --git a/man/man1/unicode_start.1 b/man/man1/unicode_start.1
index b76ccb3..59aae01 100644
--- a/man/man1/unicode_start.1
+++ b/man/man1/unicode_start.1
@@ -6,7 +6,6 @@ unicode_start \- put keyboard and console in unicode mode
 .B unicode_start
 .RI [ font " [" umap ]]
 .SH DESCRIPTION
-.IX "unicode_start command" "" "\fLunicode_start\fR command"  
 .LP
 The
 .B unicode_start
diff --git a/man/man1/unicode_stop.1 b/man/man1/unicode_stop.1
index 2fa2015..367de9b 100644
--- a/man/man1/unicode_stop.1
+++ b/man/man1/unicode_stop.1
@@ -5,7 +5,6 @@ unicode_stop \- revert keyboard and console from unicode mode
 .SH SYNOPSIS
 .B unicode_stop
 .SH DESCRIPTION
-.IX "unicode_stop command" "" "\fLunicode_stop\fR command"  
 .LP
 The
 .B unicode_stop
diff --git a/man/man1misc/splitfont.1 b/man/man1misc/splitfont.1
index 573bc3b..43a499c 100644
--- a/man/man1misc/splitfont.1
+++ b/man/man1misc/splitfont.1
@@ -11,7 +11,7 @@ splitfont
 
 .SH DESCRIPTION
 The command
-.splitfont
+.B splitfont
 removes characters from a given font file; it removes all the
 specified characters from the named font file.
 .P
diff --git a/man/man5/keymaps.5 b/man/man5/keymaps.5
index 1b28a29..0dc2c81 100644
--- a/man/man5/keymaps.5
+++ b/man/man5/keymaps.5
@@ -5,11 +5,6 @@
 .SH NAME
 keymaps \- keyboard table descriptions for loadkeys and dumpkeys
 .SH DESCRIPTION
-.IX "keymaps" "" "\fLkeymaps\fR \(em keyboard table descriptions for loadkeys and dumpkeys" ""
-.IX "loadkeys" "keyboard table descriptions" "\fLloadkeys\fR" "keyboard table descriptions"
-.IX "dumpkeys" "keyboard table descriptions" "\fLdumpkeys\fR" "keyboard table descriptions"
-.IX keyboard "table descriptions for loadkeys and dumpkeys" keyboard "table descriptions for \fLloadkeys\fR and \fLdumpkeys\fR"
-.IX "translation tables"
 .LP
 These files are used by
 .BR loadkeys (1)
diff --git a/man/man8/setfont.8.in b/man/man8/setfont.8.in
index 5ae63dd..645433b 100644
--- a/man/man8/setfont.8.in
+++ b/man/man8/setfont.8.in
@@ -12,7 +12,6 @@ setfont \- load EGA/VGA console screen font
 .RI "[-C " "console" ]
 .RI "[-h" "H" ]
 .RI "[-v] [-V]"
-.IX "setfont command" "" "\fLsetfont\fR command"
 .SH DESCRIPTION
 The
 .B setfont
diff --git a/man/man8misc/mk_modmap.8 b/man/man8misc/mk_modmap.8
index 9db007f..782cb5f 100644
--- a/man/man8misc/mk_modmap.8
+++ b/man/man8misc/mk_modmap.8
@@ -26,7 +26,7 @@ It requires the X header files to be installed.
 (In Debian, these are in the "xlibs-dev" package).
 .SH NOTES
 Since XFree version 2.1, X initialises its keymap using the Linux
-keymap, so xmodmap will often be superflous.
+keymap, so xmodmap will often be superfluous.
 
 .SH SEE ALSO
 .BR dumpkeys (1),
@@ -39,4 +39,4 @@ keymap, so xmodmap will often be superflous.
 .\" .br
 .\" Pablo Saratxaga <pablo@mandrakesoft.com>
 .\" .br
-.\" Manpage by Alastair McKinstry, <mckinstry@computer.org>
\ No newline at end of file
+.\" Manpage by Alastair McKinstry, <mckinstry@computer.org>
diff --git a/man/man8misc/setvesablank.8 b/man/man8misc/setvesablank.8
index 8a48867..9622aac 100644
--- a/man/man8misc/setvesablank.8
+++ b/man/man8misc/setvesablank.8
@@ -16,7 +16,6 @@ setvesablank \- Turn VESA screen blanking on or off
 ]
 .LP
 .SH DESCRIPTION
-.IX "setvesablank command" "" "\fLsetvesablank\fR command"
 .LP
 The program
 .B setvesablank
-- 
tg: (cfffc72..) fixes/manpages (depends on: upstream/master)
